joblib/_dask.py

```diff
@@ -269,15 +269,22 @@ async def maybe_to_futures(args):
                     try:
                         f = call_data_futures[arg]
                     except KeyError:
+                        pass
```

Comment:
Now, we won't get confusing `KeyErrors` if something wrong happens during `scattering` :)